### DISCUSSION TITLE
Automatically download new episodes

### DISCUSSION THREAD

**Discussion by izu-co** [anidl/multi-downloader-nx#75] [General]
So I want to start with a little question:

```How are you using the programm?```

But why do I want to know that? Well I've written myself a little program that keeps track of the shows I watch and then checks and downloads new episodes (if available).
If a lot of you are using the tool in a same way, I could implement a way to do exactly that.

If you use the tool as described, just react with a 👍. If you are using it differently, comment it below!

(I mean it's all for demonstrating anyway and using the tool is not encouraged)

**Comment by OryWareKyrie** [comment]
I liked the idea manin :smile:

**Comment by stratumadev** [comment]
https://github.com/HyperNylium/mdnx-auto-dl
